pkgs/tools/text/par/default.nix

```diff
@@ -18,7 +18,12 @@ stdenv.mkDerivation {
     })
   ];
 
-  buildPhase = ''make -f protoMakefile'';
+  makefile = "protoMakefile";
+  preBuild = ''
+    makeFlagsArray=( CC="${stdenv.cc.targetPrefix}cc -c" \
```

Comment:
This allows to still add stuff to makeFlagsArray in an override:

```suggestion
    makeFlagsArray+=( CC="${stdenv.cc.targetPrefix}cc -c" \
```

Reply:
Ok, force-pushed.